src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -218,6 +218,8 @@ export const ArtworkApp: React.FC<Props> = props => {
 
       <ArtworkTopContextBarFragmentContainer artwork={artwork} />
 
+      <ArtworkAuctionCreateAlertHeaderFragmentContainer artwork={artwork} />
```

Comment:
@tam-kis I wound up moving _all_ the logic into the component, and removed the `<GridColumns />` thing as well, which i don't think was doing anything. This now looks similar to the other fragment containers here.